src/geo/cartodb-layer-group.js

```diff
@@ -142,6 +142,7 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     if (staticImageURLTemplate) {
       staticImageURLTemplate = this._appendParamsToURL(staticImageURLTemplate, [ 'layer=' + this._getIndexesOfVisibleLayers() ]);
       staticImageURLTemplate = this._appendAuthParamsToURL(staticImageURLTemplate);
+      staticImageURLTemplate = staticImageURLTemplate.replace('{s}', this.getSubdomains()[0]);
```

Comment:
Yes, we use the CDN template that Maps API returns, which can indeed include a `{s}`. This is actually the problem that we're solving with this PR.